src/atlas/CMakeLists.txt

```diff
@@ -710,6 +710,7 @@ util/KDTree.cc
 util/KDTree.h
 util/PolygonXY.cc
 util/PolygonXY.h
+util/Matrix.h
```

Comment:
Should n't this be renamed to Matrix2x2.h ?

Reply:
I've now renamed this class `SquareMatrix2` to avoid any ambiguity.